apps/admin-gui/src/app/vos/pages/vo-select-page/vo-select-page.component.ts

```diff
@@ -38,4 +47,15 @@ export class VoSelectPageComponent implements OnInit {
   applyFilter(filterValue: string) {
     this.filterValue = filterValue;
   }
+
+  onCreateVo() {
+    const dialogRef = this.dialog.open(CreateVoDialogComponent, {
+      width: '1000px',
+      data: { theme: 'vo-theme'}
+    });
+
+    dialogRef.afterClosed().subscribe(() => {
+      this.refreshTable();
```

Comment:
I dont think its necessary to reload page if the dialog was cancelled. 
` dialogRef.afterClosed().subscribe((isVoCreated) => {
      if (isVoCreated) {
             this.refreshTable()
       };`